rq/job.py

```diff
@@ -593,6 +627,7 @@ def delete(self, pipeline=None, remove_from_queue=True,
 
         connection.delete(self.key)
```

Comment:
Can you change this into:
```python
connection.delete(self.key, self.dependents_key, self.dependencies_key)
```

Reply:
In 7c690087a9f4d287d400e8b489eb851e94fedfc0"
https://github.com/rq/rq/blob/7c690087a9f4d287d400e8b489eb851e94fedfc0/rq/job.py#L625